src/main/UserActivityMonitor.ts

```diff
@@ -82,6 +82,7 @@ export class UserActivityMonitor extends EventEmitter {
    */
     stopMonitoring() {
         clearInterval(this.systemIdleTimeIntervalID);
+        this.systemIdleTimeIntervalID = -1;
```

Comment:
Is this change related to Developer Mode? Just wondering

Reply:
Not directly, it fixed an issue where if we wanted to start monitoring again, we check if the ID is <0. But we never reset it after we stopped monitoring, so it wouldn't be able to restart.